src/procedures/proc_bfs.c

```diff
@@ -140,12 +135,6 @@ static ProcedureResult Proc_BFS_Invoke
 	bfs_ctx->nodes = V;
 	bfs_ctx->parents = PI;
 
-	// matrix iterator requires matrix format to be sparse
-	// to avoid future conversion from HYPER-SPARSE, BITMAP, FULL to SPARSE
-	// we set matrix format at creation time
-	GxB_Vector_Option_set(bfs_ctx->nodes, GxB_SPARSITY_CONTROL, GxB_SPARSE);
-	GxB_Vector_Option_set(bfs_ctx->parents, GxB_SPARSITY_CONTROL, GxB_SPARSE);
```

Comment:
For our iterator to work we require the iterated matrix to be either `SPARSE` or `HYPER-SPARSE`.

Reply:
but it is created as sparse